main.py

```diff
@@ -95,7 +95,10 @@ def main():
         save_path = os.path.join(timestamp,save_path)
     save_path = os.path.join(args.dataset,save_path)
     print('=> will save everything to {}'.format(save_path))
-    os.makedirs(save_path, exist_ok=True)
+    try:
+        os.makedirs(save_path)
+    except os.error: # if directory exists
+        pass
```

Comment:
Again a python 2.7 thing i believe ? the exist_ok option is so much more elegant !
But i Guess it's fine.
Still can you make instead a if statement with `os.path.exists`
I feel like `os.error` can contain other error and passing everything seems not ideal.